addon/components/o-s-s/nav-tab.stories.js

```diff
@@ -0,0 +1,46 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::NavTab',
+  component: 'NavTab',
+  argTypes: {
+    onSelection: {
+      type: { required: true },
+      description: 'A callback triggered when the tab has been clicked.'
+    },
+
+    tabArray: {
+      type: { required: true },
+      description:
+        'Array of TabDefinition which have a parameter: <br/> -icon?: string; <br/> -label?: string; <br/> -infoCircle?: boolean; <br/> -notificationDot?: boolean; <br/> -selected: boolean; <br/> -disabled: boolean; <br/> @label or @icon is mandatory for each element of tabArray'
+    }
+  }
+};
+
+const defaultArgs = {
+  tabArray: [
+    { label: 'Tab', icon: 'far fa-thumbs-up' },
+    { label: 'Tab2', icon: 'far fa-thumbs-up' },
+    { label: 'Tab3' },
+    { icon: 'far fa-thumbs-up' },
+    { label: 'Tab', icon: 'far fa-thumbs-up', infoCircle: true },
+    { label: 'Tab', icon: 'far fa-thumbs-up', infoCircle: true, notificationDot: true },
+    { label: 'Tab', icon: 'far fa-thumbs-up', infoCircle: true, notificationDot: true, selected: true },
+    { label: 'Tab', icon: 'far fa-thumbs-up', infoCircle: true, notificationDot: true, disabled: true },
+    { label: 'Tab', icon: 'far fa-thumbs-up', infoCircle: true, notificationDot: true, selected: true, disabled: true }
```

Comment:
Yes but I put this in tab because we can see that an element selected true and disable true take the disable style.
And this case shouldn't exist too with selected true and disable true.
So are you sure that i delete it ?

Reply:
Indeed ! thanks for the explanation :)